packages/contracts-bedrock/src/legacy/AddressManager.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity 0.8.15;
 
 import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
 
-/// @custom:legacy
+/// @custom:legacy true
```

Comment:
`@custom:legacy` had to be turned into `@custom:legacy true` to avoid eating the line below.